sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/CertificateClient.java

```diff
@@ -1650,10 +1983,42 @@ public KeyVaultCertificateWithPolicy importCertificate(ImportCertificateOptions
      * @param importCertificateOptions The details of the certificate to import to the key vault
      * @param context Additional context that is passed through the Http pipeline during the service call.
      * @throws HttpRequestException when the {@code importCertificateOptions} are invalid.
-     * @return A {@link Response} whose {@link Response#getValue() value} contains the {@link KeyVaultCertificateWithPolicy imported certificate}.
+     * @return A {@link Response} whose {@link Response#getValue() value} contains the
```

Comment:
Missing throws statement.
```suggestion
     * @throws NullPointerException if {@code importCertificateOptions} is null.
     * @return A {@link Response} whose {@link Response#getValue() value} contains the
```

Reply:
Also found it's missing from `importCertificate(ImportCertificateOptions)`, but GitHub won't let me add a code suggestion there as no code was changed in that block.